.github/workflows/auto.yml

```diff
@@ -0,0 +1,35 @@
+name: Automatic Build
+
+on:
+  push:
+
+jobs:
+  deploy:
```

Comment:
I know you borrowed this from another CI, where we are actually deploying this onto a website; but given that this is not an actual deployment, I would prefer another name for this instead of `deploy`, like "build" or any other suggestion you may have

Reply:
fixed in [2c8b14a](https://github.com/unicef/publicgoods-scripts/commit/2c8b14a532729966787dfd255d7b304a6ec3f7b9)